docs/content/preview/yugabyte-platform/configure-yugabyte-platform/alert-policy-templates.md

```diff
@@ -0,0 +1,306 @@
+---
+title: Alert policy templates
+headerTitle: Alert policy templates
+linkTitle: Alert policy templates
+description: Alert policy template reference
+menu:
+  preview_yugabyte-platform:
+    identifier: alert-policy-templates
+    parent: set-up-alerts-health-checking
+    weight: 40
+type: docs
+---
+
+Alert policies use templates to define how the alert is triggered. The alert templates have been created using Prometheus expressions.
+
```

Comment:
Will be useful to mention where you can see these policy templates from the UI or even link to [create-alerts](https://deploy-preview-17128--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check/#create-alerts) to find this list of templates.